go-controller/pkg/ovn/egressip.go

```diff
@@ -164,11 +164,15 @@ func (oc *Controller) reconcileEgressIP(old, new *egressipv1.EgressIP) (err erro
 		}
 		if len(ipsToAdd) > 0 {
 			statusToAdd = oc.assignEgressIPs(name, ipsToAdd.List())
-			if err := oc.addEgressIPAssignments(name, statusToAdd, newEIP.Spec.NamespaceSelector, newEIP.Spec.PodSelector); err != nil {
-				return err
-			}
 			statusToKeep = append(statusToKeep, statusToAdd...)
 		}
+		// Assign all statusToKeep, we need to warm up the podAssignment cache
+		// on restart. We won't perform any additional transactions to the NB DB
+		// for things which exists because the libovsdb operations use
+		// modelClient which is idempotent.
+		if err := oc.addEgressIPAssignments(name, statusToKeep, newEIP.Spec.NamespaceSelector, newEIP.Spec.PodSelector); err != nil {
```

Comment:
should this commit be a fixup/squashed to your previous commit where you fix cache sync on start up?

Reply:
Yeah, will fix that.